File: RoutineSchedulerFrontend/src/models/UpdateScheduleDto.js
export default class UpdateScheduleDto {
    constructor(scheduleId, teacherId, batchId, subjectId, day, timeSlot) {
      this.scheduleId = scheduleId;
      this.teacherId = teacherId;
      this.batchId = batchId;
      this.subjectId = subjectId;
      this.day = day;
      this.timeSlot = timeSlot;
    }
  }
  

File: API/RMSAPI/Controllers/DTO/UpdateScheduleDTO.cs
namespace RMSAPI.Controllers.DTO;

public class UpdateScheduleDto
{
    public int ScheduleId { get; set; }
    public int BatchId { get; set; }
    public DayOfWeek Day { get; set; }
    public TimeSpan StartTime { get; set; }
    public TimeSpan EndTime { get; set; }
    public int SubjectId { get; set; }
    public int TeacherId { get; set; }
}
